src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -837,3 +848,152 @@ class CacheDictionaryDeleteResponseError extends CacheDictionaryDeleteResponse
 {
     use ErrorBody;
 }
+
+abstract class CacheDictionaryFetchResponse extends ResponseBase
+{
+    public function asHit(): CacheDictionaryFetchResponseHit|null
+    {
+        if ($this->isHit()) {
+            return $this;
+        }
+        return null;
+    }
+
+    public function asMiss(): CacheDictionaryFetchResponseMiss|null
+    {
+        if ($this->isMiss()) {
+            return $this;
+        }
+        return null;
+    }
+
+    public function asError(): CacheDictionaryFetchResponseError|null
+    {
+        if ($this->isError()) {
+            return $this;
+        }
+        return null;
+    }
+}
+
+class CacheDictionaryFetchResponseHit extends CacheDictionaryFetchResponse
+{
+    private RepeatedField $items;
```

Comment:
Is there any reason to hold onto these `items`?

Reply:
Nope! Thank you for pointing it out.